<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportIfcZipDialog</class>
 <widget class="QDialog" name="ExportIfcZipDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>463</width>
    <height>132</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>IfcAlignment BuildingSmart P6 Step File Export</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../Resources/OpenInfraPlatform.qrc">
    <normaloff>:/icons/icon.ico</normaloff>:/icons/icon.ico</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QCheckBox" name="checkBoxExportAlignment">
     <property name="text">
      <string>Export alignment</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QPushButton" name="pushButtonCancel">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QCheckBox" name="checkBoxExportDigitalElevationModel">
     <property name="text">
      <string>Export digital elevation model</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QPushButton" name="pushButtonExport">
     <property name="text">
      <string>Export</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Angle measurement</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxAngleMeasurement">
       <property name="currentText">
        <string>Radiant</string>
       </property>
       <item>
        <property name="text">
         <string>Radiant</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Degree</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../Resources/OpenInfraPlatform.qrc"/>
 </resources>
 <connections/>
</ui>
